cpp/generate.rb

```diff
@@ -55,6 +55,29 @@
             tag_as: "storage.modifier.reference"
         ).maybe(@spaces)
     )
+    functionCallTemplate = ->(repository_name:nil, match_name: nil, tag_name_as: nil, tag_content_as: nil, tag_parenthese_as: nil) do
```

Comment:
this should be renamed to not include the word template. maybe `generateFunctionCall` ?

Reply:
yeah, and we might actually want to rename the `Finders` to `Generate` since they're kind of in the same space.